tests/config/BUILD

```diff
@@ -27,6 +27,9 @@ load(
     "gcc_env",
 )
 
+bazel_latest = "0.19.0"
+bazel_latest_rc = "1"
```

Comment:
This is confusing. What if the latest release is 0.20.0, and the latest rc is 0.21.0 rc2. What do we put in bazel_latest and bazel_latest_rc?